README.md

```diff
@@ -55,16 +63,21 @@ yarn add @lit-protocol/lit-node-client
 
 # Packages
 
-> 📝 If you're looking to use the Lit SDK, you're probably all set with just the lit-node-client <link>. It's got everything most folks need. However, if you're the type who likes to keep things super lean and mean, and you need to keep an eagle eye on your bundle size, we've got individual packages too.
-
-Individual packages are below so that you can import the minimum required packages for your use case.
+📝 If you're looking to use the Lit SDK, you're probably all set with just the lit-node-client <link>. It's got everything most folks need. 
```

Comment:
maybe rephrase the sentence along the lines of "get started with interacting with lit network"